api/v1/topologymap.go

```diff
@@ -76,3 +79,19 @@ func (m *NodeTopologyMap) GetKeyValues(topologyKey string) (string, []string) {
 
 	return topologyKey, values
 }
+
+// GetKeyValues returns a node label matching the topologyKey and all values for StrechCluster
+// for that label across all storage nodes
+func (m *NodeTopologyMap) GetKeyValuesStrechCluster() (string, []string) {
+	values := []string{}
+	topologyKey := ""
+	for label, labelValues := range m.Labels {
+		if label == corev1.LabelZoneFailureDomainStable || label == labelZoneFailureDomainWithoutBeta {
+			topologyKey = label
+			values = labelValues
+			break
+		}
+	}
```

Comment:
This means for stretch cluster we can't have a different failure zone? Like, `rack`, `host`?

Reply:
We should update the `GetKeysValues` function to do an equality check with valid labels that we define in the `validTopologyLabelKeys` and return the key, values accordingly.

Reply:
@malayparida2000 @iamniting what do you guys think?

Reply:
Stretch always uses zones, just to confirm in this thread in addition to comments from others that confirmed the same.

Reply:
done